### ISSUE TITLE
New error message format in gfortran 5

### ISSUE THREAD

**Issue by azag0** [scrooloose/syntastic#1465]
The format of error messages in gfortran 5 changed by one character. The line and column number are now separated by a colon, not a period. To support both versions <5 and 5, [this line](https://github.com/scrooloose/syntastic/blob/b1b79009e3df39d7e66c06f5dafdeaf8a3594b52/syntax_checkers/fortran/gfortran.vim#L47) needs to be changed to
```viml
            \ '%A%f:%l\[.:\]%c:,'.
```

**Comment by azag0** [comment]
Just noticed there are also fatal errors in gfortran 5. The following line captures those,
```viml
            \ '%ZFatal %trror: %m,'.
```

**Comment by lcd047** [comment]
Thank you.  Fixed in commit a596ef3.
